### PR TITLE
Add team wall mode

### PR DESCRIPTION
**Pull request by szymonkozak**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -13,7 +15,7 @@ android {
     defaultConfig {
         applicationId "pl.temomuko.autostoprace"
         minSdkVersion 16
-        targetSdkVersion 25
+        targetSdkVersion 27
```
Comment (freszu) [inline app/build.gradle]: back to 25, we didn't test our services on oreo

File: app/src/main/java/pl/temomuko/autostoprace/ui/teamslocationsmap/adapter/wall/WallAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val wallItem = wallItems[position]
+        holder.bind(wallItem)
+    }
+
+    override fun getItemCount() = wallItems.size
+
+    inner class ViewHolder(override val containerView: View) : RecyclerView.ViewHolder(containerView), LayoutContainer {
```
Comment (freszu) [inline app/src/main/java/pl/temomuko/autostoprace/ui/teamslocationsmap/adapter/wall/WallAdapter.kt]: to separate file

File: app/src/main/res/layout/activity_teams_locations_map.xml [hunk 3]
```diff
@@ -64,6 +65,41 @@
...
+                    android:background="@color/standard_background_color">
+
+                    <LinearLayout
+                        android:id="@+id/ll_empty_state"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:gravity="center"
+                        android:orientation="vertical"
+                        android:padding="32dp">
```
Comment (freszu) [inline app/src/main/res/layout/activity_teams_locations_map.xml]: dont we have some generic small, medium, big paddings?

File: app/src/main/res/layout/item_wall.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:orientation="vertical"
+              android:paddingEnd="4dp"
```
Comment (freszu) [inline app/src/main/res/layout/item_wall.xml]: as above

File: app/src/main/res/layout/item_wall.xml [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+            <android.support.constraint.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <ImageView
+                    android:id="@+id/photoImageView"
+                    android:layout_width="0dp"
+                    android:layout_height="0dp"
+                    android:layout_marginEnd="0dp"
```
Comment (freszu) [inline app/src/main/res/layout/item_wall.xml]: remove those 0 margins

File: app/src/main/res/layout/item_wall.xml [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                android:padding="16dp">
+
+                <TextView
+                    android:id="@+id/messageTextView"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="12dp"
+                    android:lineSpacingExtra="1dp"
+                    android:textColor="@color/wall_message"
+                    android:textSize="15sp"
```
Comment (freszu) [inline app/src/main/res/layout/item_wall.xml:54]: cant some standard text styles be used in this file?

File: app/src/main/res/layout/item_wall.xml [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                    tools:src="@drawable/bg"/>
+
+            </android.support.constraint.ConstraintLayout>
+
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:padding="16dp">
```
Comment (freszu) [inline app/src/main/res/layout/item_wall.xml]: again paddings

**Review by freszu — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/item_wall.xml (hunk 6 above)
Comment (szymonkozak) [reply app/src/main/res/layout/item_wall.xml:54]: No

Comment (freszu) [reply app/src/main/res/layout/item_wall.xml:54]: 😞

**Review by freszu — APPROVED** [review]

**Pull request merged**